tests/extensions/test_comment_ending.py

```diff
@@ -0,0 +1,35 @@
+import os.path as osp
+
+import pytest
+
+import pylint.extensions.comment_ending as comment_ending
+
+
+@pytest.fixture(scope="module")
+def checker():
+    return comment_ending.CommentChecker
+
+
+@pytest.fixture(scope="module")
+def enable():
+    return ["empty-comment-ending"]
+
+
+@pytest.fixture(scope="module")
+def disable():
+    return ["all"]
+
+
+def test_comment_base_case(linter):
+    comment_test = osp.join(
+        osp.dirname(osp.abspath(__file__)), "data", "comment_ending.py"
```

Comment:
Can you use pathlib.Path instead ? I know the suggestion is not working as it is, it's just an illustration :) 
```suggestion
       from pathlib import Path
       Path(__file__).parent / "data" / "comment_ending.py"
```

Reply:
fixed in `python3.5 fix` and `PR review` commits